samples/stable-diffusion/manifests/cert-manager.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: letsencrypt-staging
+spec:
+  acme:
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    email: jirka@kedify.io
```

Comment:
if we have any better email, I would put it here (and line 23). It should send an email 1 month before the expiration